src/server/utilities/cspHeader/index.js

```diff
@@ -0,0 +1,157 @@
+export const generateScriptSrc = ({ isAmp, isLive }) => {
+  if (isAmp) {
```

Comment:
This procedural setup is what was in the initial file: https://github.com/bbc/simorgh/pull/6053/files#diff-3214365de34a06e10158f7412e186dd3L13 
Now that I have the majority of the setup completed, I am going to refactor this and have a simple object with values that are provided to the relevant function generator. This will be more readable and easier to maintain.
e.g.
```
const directives = {
  connectSrc: {
    ampLive: [
      'https://*.akstat.io',
       ...
    ],
    canonicalLive: [],
    ampNonLive: [],
    canonicalNonLive: [],
  },
  frameSrc: {
    ampLive: [],
    canonicalLive: [],
    ampNonLive: [],
    canonicalNonLive: [],
  },
  ...
};
```

Reply:
This has now been refactored entirely.